src/ansys/fluent/core/services/field_data.py

```diff
@@ -638,7 +639,55 @@ class PathlinesFieldDataRequest(NamedTuple):
     zones: list | None = None
 
 
-class BaseFieldData:
+class FieldDataSource(ABC):
```

Comment:
@prmukherj If you're making the transaction part abstract as well then it would follow that transaction creation appears in this interface.

Reply:
Actually the 'FieldDataSource' is used as abstract class for both the Field Data output as well as the Field Data output from transaction. The later has no implementation of 'new_transaction'.

Reply:
Done, Thank you